cypress/e2e/dashboard.spec.ts

```diff
@@ -0,0 +1,270 @@
+        E2E_EMAIL_TEST_SITE.repo
+      )
+
+      // Act
+      getReviewRequestButton().click()
+      cy.contains(REVIEW_MODAL_SUBTITLE).should("be.visible")
+      cy.get(TITLE_INPUT_SELECTOR).type(MOCK_REVIEW_TITLE)
+      // NOTE: should not allow submitting of review prior to selecting reviewer
+      getSubmitReviewButton().should("be.disabled")
+      getAddReviewerDropdown().click()
+      // select reviewer and click
+      selectReviewer(E2E_EMAIL_COLLAB.email)
+      removeReviewers()
+      getSubmitReviewButton().should("be.disabled")
+      // add back reviewer
+      getAddReviewerDropdown().click()
+      selectReviewer(E2E_EMAIL_COLLAB.email)
+      cy.get(DESCRIPTION_TEXTAREA_SELECTOR).type(MOCK_REVIEW_DESCRIPTION)
+
+      // Assert
+      getSubmitReviewButton().click()
+      cy.contains("Review request submitted").should("be.visible")
+    })
+  })
+  describe.skip("has pending review requests", () => {
+    before(() => {
+      addAdmin(E2E_EMAIL_COLLAB.email)
+      // NOTE: Create a review request if none exists
+      listReviewRequests().then((requests) => {
+        if (!requests || !requests.length) {
+          createReviewRequest(
+            MOCK_REVIEW_TITLE,
+            [E2E_EMAIL_COLLAB.email],
+            MOCK_REVIEW_DESCRIPTION
+          )
+        }
+      })
+    })
+    it("should have the pending review alert on the workspace", () => {
+      // Arrange
+      // Act
+      // Assert
+    })
+    it("should prevent the requestor from approving their own review request", () => {
+      // Arrange
+      // Act
+      // Assert
+    })
+    it('should show the review request on the workspace with the tag "Pending review"', () => {
+      // Arrange
+      // Act
+      // Assert
+    })
+    it("should have changes reflected in the currently open review request", () => {
+      // Arrange
+      // TODO: If this is hard, maybe we can just do delete + add content + have new page
+      // NOTE: We define changes as being
+      // 1. Adding a new page
+      // 2. Editing an existing page
+      // 3. Deleting an existing page
+      // 4. Rename an existing page
+      // 5. Moving an existing page
+      // Act
+      // Assert
+    })
+    it("should be able to have the request approved as a collaborator", () => {
+      // Arrange
+      cy.setEmailSessionDefaults("Email collaborator")
+      // Act
+      // Assert
+    })
+  })
+  describe.skip("has approved review requests", () => {
+    before(() => {
+      addAdmin(E2E_EMAIL_COLLAB.email)
+      // NOTE: Create a review request if none exists
+      listReviewRequests().then((requests) => {
+        if (!requests || !requests.length) {
+          createReviewRequest(
+            MOCK_REVIEW_TITLE,
+            [E2E_EMAIL_COLLAB.email],
+            MOCK_REVIEW_DESCRIPTION
+          ).then((id) => approveReviewRequest(id))
+        }
+      })
+    })
+    it("should have the alert on the workspace", () => {
```

Comment:
can we rename the phrase 'the alert'? 
something liek 
"should have an alert stating that editing is disabled when accessing workspace"

Reply:
sure - i think `review request alert` might be more descriptive in context. will update to your desc + the phrase

Reply:
done